File: 프로그래머스/0/120883. 로그인 성공？/로그인 성공？.js
function solution(id_pw, db) {
    var answer = '';
    const map = new Map(db);
    if(map.has(id_pw[0])){
        if(map.get(id_pw[0]) === id_pw[1]){
            return 'login'
        }
        return 'wrong pw'
    }
    return 'fail';
}

File: 프로그래머스/0/120883. 로그인 성공？/로그인 성공？.py
def solution(id_pw, db):
    answer = ''
    u_id, u_pw = id_pw
    db_dict = dict(db)
    
    if u_id not in db_dict:
        return 'fail'
        
    return 'login' if db_dict[u_id] == u_pw else 'wrong pw'